### DISCUSSION TITLE
Server-side Phoenix

### DISCUSSION THREAD

**Discussion by contrebande-labs** [ACINQ/phoenix#273] [Improving Phoenix]
Hello,

We would be interested in using Phoenix a little bit like Bitrefill's Thor incoming channel capacity service. That is we want to run only one "master" eclair node in one of our datacenters and if that node is not reachable from the datacenter hosting the POS where we want to receive a payment, then we can spin off a fallback "lite" node like Phoenix that buys incoming capacity from ACINQ or Bitrefill.

Would that be a use case the ACINQ team would be willing to support?

**Comment by pm47** [comment]
Hi,

I'm not completely sure I understand what you mean. We have indeed thought about having a Phoenix running on a server instead of a mobile phone, with an API instead of a UI. Basically it would behave like a regular LN node, but with automated inbound liquidity management. Is that what you have in mind?

**Reply by contrebande-labs** [reply]
Yes, exactly. Running on the JVM. Using a custom database backend for the cloud (Cassandra, DynamoDB, Oracle NoSQL, etc.).

**Comment by contrebande-labs** [comment]
Well, I am not familiar with Kotlin or Gradle or Android so would it be possible to have the `phoenix-shared` module to be maven-publishable like `lightning-kmp` is (ie with `:publishToMavenLocal`) ? And then I could try poking around the `PhoenixBusiness` class... Thanks.
